source/reviews/comments.txt

```diff
@@ -0,0 +1,112 @@
+==========================================
+How To Write Code And Copy Review Comments
+==========================================
+
+Summary
+-------
+
+- Be kind.
+- Explain your reasoning.
+- Balance giving explicit directions with just pointing out problems and letting
+  the author decide.
+- Encourage authors to simplify code or add code comments instead
```

Comment:
Comment: This seems to go against guideline of "don't include code comments" I've received throughout reviews.

```suggestion
- Encourage authors to simplify code instead
```